app/controllers/participate/survey/index.js

```diff
@@ -1,10 +1,30 @@
 import Ember from 'ember';
+import ENV from 'isp/config/environment';
+
 
 export default Ember.Controller.extend({
   model: null,
   session: null,
-  frameIndex: 0,
-  framePage: 0,
+  frameIndex: Ember.computed(function() {
+    var frameIndex = 0;
+    if (ENV.continueSession) {
+      var session = this.get('session');
+      if (session && session.get('frameIndex')) {
+        frameIndex = session.get('frameIndex');
+      }
+    }
+    return frameIndex;
+  }),
+  framePage: Ember.computed(function() {
+    var framePage = 0;
+    if (ENV.continueSession) {
+      var session = this.get('session');
+      if (session && session.get('framePage')) {
```

Comment:
Do you have any suggestions on what to call it? The `frame` is the survey section they're on so I thought `framePage` would be consistent